==41731== Invalid read of size 8
==41731==    at 0x10A07D: cadenaSiguiente(_rep_cadena*) (cadena.cpp:109)
==41731==    by 0x10A1BE: cadenaDeColCadenas(unsigned int, _rep_colCadenas*) (colCadenas.cpp:31)
==41731==    by 0x109A8E: main (principal.cpp:178)
==41731==  Address 0x10 is not stack'd, malloc'd or (recently) free'd
==41731== 
==41731== 
==41731== Process terminating with default action of signal 11 (SIGSEGV)
==41731==  Access not within mapped region at address 0x10
==41731==    at 0x10A07D: cadenaSiguiente(_rep_cadena*) (cadena.cpp:109)
==41731==    by 0x10A1BE: cadenaDeColCadenas(unsigned int, _rep_colCadenas*) (colCadenas.cpp:31)
==41731==    by 0x109A8E: main (principal.cpp:178)
==41731==  If you believe this happened as a result of a stack
==41731==  overflow in your program's main thread (unlikely but
==41731==  possible), you can try to increase the size of the
==41731==  main thread stack using the --main-stacksize= flag.
==41731==  The main thread stack size used in this run was 8388608.
1>#  implementar cadenaDeColCadenas.
2>La cadena 0 es :timeout: la orden monitorizada ha volcado un 'core'
Segmentation fault
